components/post_view/post_info/post_info.test.tsx

```diff
@@ -40,12 +40,18 @@ describe('components/post_view/PostInfo', () => {
         shortcutReactToLastPostEmittedFrom: '',
         actions: {
             removePost: jest.fn(),
+            setActionsMenuInitialisationState: jest.fn(),
         },
         shouldShowDotMenu: true,
+        firstTimeActionsMenuOpened: false,
+        shouldShowActionsMenu: true,
         isReadOnly: true,
         collapsedThreadsEnabled: false,
     };
 
+    // test('should not show actions menu, plugins disabled', () => {
```

Comment:
Leaving a not here in to confirm whether this test will be implemented or these comments will be removed.